### PR TITLE
Automation COMMERCE-10892/11735

### PR DESCRIPTION
**Pull request by 22LucaGiannotta22**

### CODE REVIEW

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceAccountSupplierRole.testcase [hunk 1]
```diff
@@ -15,6 +15,58 @@ definition {
...
+
+		task ("Then the admin can assert that the catalog is successfully linked to the account using GET API") {
+			var accountId = CommerceJSONCatalogsAPI._getCommerceAccountIdByCatalogId(catalogName = "Supplier Catalog");
+
+			TestUtils.assertEquals(
+				actual = ${accountId},
+				expected = ${accountEntryId});
+		}
+
+		task ("When the admin changes the account name") {
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceAccountSupplierRole.testcase:47]: here it should be catalog:
When the admin changes the catalog name

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceAccountSupplierRole.testcase [hunk 2]
```diff
@@ -15,6 +15,58 @@ definition {
 		CommerceConfiguration.commerceTearDown();
 	}
 
+	@description = "This is a test case for COMMERCE-11825. Admin can verify that supplier account Id is right even after changing the catalog's name"
```
Comment (baharturk) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceAccountSupplierRole.testcase:18]: Admin can verify that changing the catalog's name does not affect the supplier account Id and it remains the same.

**Review by baharturk — CHANGES_REQUESTED** [review]
Please have a look at my comments. Thanks

**Comment by 22LucaGiannotta22** [comment]
ci:test:sf

**Comment by 22LucaGiannotta22** [comment]
ci:forward

**Pull request closed without merging**